spec/services/course_exports/prepare_teams_export_service_spec.rb

```diff
@@ -97,7 +97,11 @@
 
   let!(:school_admin) { create :school_admin, school: school }
   let(:course_export) do
-    create :course_export, :teams, course: course, user: school_admin.user
+    create :course_export,
+           :teams,
+           course: course,
+           cohorts: [cohort_live],
+           user: school_admin.user
```

Comment:
The spec doesn't check for cohort scoping; Have more than one cohort on the list to ensure that the export works as expected.

Ideally have 3 cohorts, pass 2 to the export